will/requirements/base.txt

```diff
@@ -1,28 +1,29 @@
 APScheduler==2.1.2
 beautifulsoup4==4.6.0
```

Comment:
Regarding updating dependencies. beautifulsoup4 is only used by markdownify to my knowledge which is in the slack backend for rendering html. Do we want to update markdownify to the latest that declares 3.x support and upgrade beautifulsoup4 to the appropriate dependent version?

Reply:
I’m all for updating anything and everything to the latest version that works.